tkltest/generate/unit/augment.py

```diff
@@ -220,10 +220,7 @@ def __compute_base_and_augmenting_tests_coverage(ctd_test_dir, evosuite_test_dir
                                       raw_cov_data_dir=raw_cov_data_dir, jdk_path=jdk_path)
         if not test_coverage:
             tkltest_status('Error while computing coverage for test: {}'.format(test), error=True)
```

Comment:
Add 'Warning' to the message and also indicate we are skipping the test

Reply:
it is set as error:
error=True
do you want it to be downgrade to warning?

Reply:
It is set as error because in the original code treated it as an error and existed. After your fix we are no longer existing so now it is a warning. It can say: `Warning: encountered an error while computing coverage for test: {}, skipping it'.format(test)`